Riot/Managers/Theme/Themes/DarkTheme.swift

```diff
@@ -130,32 +130,26 @@ class DarkTheme: NSObject, Theme {
             let appearance = UINavigationBarAppearance()
             
             if BuildSettings.newAppLayoutEnabled {
-                appearance.configureWithDefaultBackground()
+                appearance.configureWithTransparentBackground()
+                appearance.backgroundEffect = UIBlurEffect(style: .dark)
             } else {
                 appearance.configureWithOpaqueBackground()
                 appearance.backgroundColor = baseColor
             }
 
-            appearance.backgroundColor = baseColor
             if !modernScrollEdgeAppearance {
                 appearance.shadowColor = nil
             }
             appearance.titleTextAttributes = [
-                NSAttributedString.Key.foregroundColor: textPrimaryColor
+                .foregroundColor: textPrimaryColor
             ]
-            
+            appearance.largeTitleTextAttributes = [
+                .foregroundColor: textPrimaryColor
+            ]
+
             navigationBar.standardAppearance = appearance
-            
-            if BuildSettings.newAppLayoutEnabled {
-                 appearance.configureWithOpaqueBackground()
-                 appearance.backgroundColor = baseColor
-                 appearance.shadowColor = nil
-                 appearance.titleTextAttributes = [
-                     NSAttributedString.Key.foregroundColor: textPrimaryColor
-                 ]
-             }
-             navigationBar.scrollEdgeAppearance = modernScrollEdgeAppearance ? nil : appearance
-            } else {
+            navigationBar.scrollEdgeAppearance = modernScrollEdgeAppearance && !BuildSettings.newAppLayoutEnabled ? nil : appearance
```

Comment:
I *think* this is meant to be:
```suggestion
            navigationBar.scrollEdgeAppearance = modernScrollEdgeAppearance || BuildSettings.newAppLayoutEnabled ? nil : appearance
```

Reply:
Here is what happens if the `navigationBar.scrollEdgeAppearance` is not set (note the example is with the iOS light theme and the app auto theme):

![ezgif-4-5d960cdef7](https://user-images.githubusercontent.com/15386762/182957105-bc1baf6f-ec79-4186-ad1d-8168985cfe28.gif)

Reply:
Beautiful 😂